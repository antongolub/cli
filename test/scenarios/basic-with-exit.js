'use strict'
require('../..')(main)

function main () {
  console.log('started')
  return new Promise((resolve, reject) => {
    process.exit()
    console.log('finished')
    resolve()
  })
}
