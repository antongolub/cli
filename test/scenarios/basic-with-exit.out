started
Abnormal exit: Promises not resolved
1
